src/bucket.js

```diff
@@ -2297,20 +2297,21 @@ class Bucket extends common.ServiceObject {
     if (is.boolean(options.resumable)) {
       upload();
     } else if (isURL) {
-      request.head(requestOptions, (err, resp) => {
-        if (err) {
-          callback(err);
-          return;
-        }
-
-        const contentLength = resp.headers['content-length'];
-
-        if (is.number(contentLength)) {
-          options.resumable = contentLength > RESUMABLE_THRESHOLD;
-        }
+      const url = requestOptions.url;
+      const method = 'HEAD';
+      const headers = requestOptions.headers;
+      const redirect = requestOptions.followAllRedirects ? 'follow' : 'manual';
```

Comment:
I think `request` follows all redirects by default.  Would it make sense to always use `follow` here?

Reply:
Good catch, I'll use it as the default.